packages/manager/modules/octavia-load-balancer/src/create/routing.js

```diff
@@ -87,6 +92,19 @@ export default /* @ngInject */ ($stateProvider) => {
         `${TRACKING_CHAPTER_1}::${TRACKING_NAME}::goto-product-page`,
       trackingRegionAvailability: () =>
         `${TRACKING_CHAPTER_1}::${TRACKING_NAME}::goto-region-availability`,
+      trackingPrivateNetworkCreation: () =>
+        `${TRACKING_CHAPTER_1}::${TRACKING_NAME}::create-private-network`,
```

Comment:
**suggestion:** Since this is all static information, it can be maintained in constant file.